pages/custom_blocks.py

```diff
@@ -196,10 +200,10 @@ class Meta:
 
 
 class FAQBlock(blocks.StructBlock):
-    question = blocks.RichTextBlock(required=True)
-    slug = blocks.CharBlock(required=True)
-    answer = blocks.RichTextBlock(required=True)
-    document = DocumentChooserBlock(required=False)
+    question = blocks.RichTextBlock(required=True, help_text='The visible text of the question (does not collapse).')
+    slug = blocks.CharBlock(required=True, help_text='Not visible to user, must be unique in this FAQ.')
+    answer = blocks.RichTextBlock(required=True, help_text='The answer to the question, is hidden until the question is expanded.')
+    document = DocumentChooserBlock(required=False, help_text='Not sure this does anything.')
```

Comment:
this originally was so you could link to a PDF, but yeah, not sure if it still works